examples/resources/cloudflare_healthcheck/import.sh

```diff
@@ -0,0 +1,5 @@
+# Use the Zone ID and Healthcheck ID to import.
+$ terraform import cloudflare_healthcheck.example 9a7806061c88ada191ed06f989cc3dac/699d98642c564d2e855e9661899b7252
```

Comment:
updating this can then remove the rest of the hints in here. 
```suggestion
$ terraform import cloudflare_healthcheck.example <zone_id>/<healthcheck_id>
```